### PR TITLE
[JENKINS-65398] Terminology update

### PR DESCRIPTION
**Pull request by twasyl**
The changes introduce terminology updates to replace `master` by `controller` and remaining `slave` by `agent`.
This is lead by [JENKINS-65398](https://issues.jenkins.io/browse/JENKINS-65398).

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your master branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

**Review by aHenryJard — APPROVED** [review]

**Review by jvz — APPROVED** [review]

File: src/main/resources/org/jenkinsci/plugins/credentialsbinding/impl/CertificateMultiBinding/help.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
     Sets one variable to the username and one variable to the password given in the credentials.
 </div>
 <div>
-    <strong>Warning</strong>: if the master or agent node has multiple executors,
+    <strong>Warning</strong>: if the controller or agent node has multiple executors,
```
Comment (rsandell) [inline src/main/resources/org/jenkinsci/plugins/credentialsbinding/impl/CertificateMultiBinding/help.html:5]:
```suggestion
    <strong>Warning</strong>: if the Jenkins controller or agent node has multiple executors,
```

File: src/main/resources/org/jenkinsci/plugins/credentialsbinding/impl/FileBinding/help.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
     (The file is deleted when the build completes.)
 </div>
 <div>
-    <strong>Warning</strong>: if the master or agent node has multiple executors,
+    <strong>Warning</strong>: if the controller or agent node has multiple executors,
```
Comment (rsandell) [inline src/main/resources/org/jenkinsci/plugins/credentialsbinding/impl/FileBinding/help.html:6]:
```suggestion
    <strong>Warning</strong>: if the Jenkins controller or agent node has multiple executors,
```

File: src/main/resources/org/jenkinsci/plugins/credentialsbinding/impl/SSHUserPrivateKeyBinding/help.html [hunk 3]
```diff
@@ -3,7 +3,7 @@
     (The file is deleted when the build completes.) Also optionally sets variables for the SSH key's username and passphrase.
 </div>
 <div>
-    <strong>Warning</strong>: if the master or agent node has multiple executors,
+    <strong>Warning</strong>: if the controller or agent node has multiple executors,
```
Comment (rsandell) [inline src/main/resources/org/jenkinsci/plugins/credentialsbinding/impl/SSHUserPrivateKeyBinding/help.html:6]:
```suggestion
    <strong>Warning</strong>: if the Jenkins controller or agent node has multiple executors,
```

File: src/main/resources/org/jenkinsci/plugins/credentialsbinding/impl/StringBinding/help.html [hunk 4]
```diff
@@ -2,7 +2,7 @@
     Sets a variable to the text given in the credentials.
 </div>
 <div>
-    <strong>Warning</strong>: if the master or agent node has multiple executors,
+    <strong>Warning</strong>: if the controller or agent node has multiple executors,
```
Comment (rsandell) [inline src/main/resources/org/jenkinsci/plugins/credentialsbinding/impl/StringBinding/help.html:5]:
```suggestion
    <strong>Warning</strong>: if the Jenkins controller or agent node has multiple executors,
```

File: src/main/resources/org/jenkinsci/plugins/credentialsbinding/impl/UsernamePasswordBinding/help.html [hunk 5]
```diff
@@ -2,7 +2,7 @@
     Sets a variable to the username and password given in the credentials, separated by a colon (<code>:</code>).
 </div>
 <div>
-    <strong>Warning</strong>: if the master or agent node has multiple executors,
+    <strong>Warning</strong>: if the controller or agent node has multiple executors,
```
Comment (rsandell) [inline src/main/resources/org/jenkinsci/plugins/credentialsbinding/impl/UsernamePasswordBinding/help.html:5]:
```suggestion
    <strong>Warning</strong>: if the Jenkins controller or agent node has multiple executors,
```

File: src/main/resources/org/jenkinsci/plugins/credentialsbinding/impl/UsernamePasswordMultiBinding/help.html [hunk 6]
```diff
@@ -2,7 +2,7 @@
     Sets one variable to the username and one variable to the password given in the credentials.
 </div>
 <div>
-    <strong>Warning</strong>: if the master or agent node has multiple executors,
+    <strong>Warning</strong>: if the controller or agent node has multiple executors,
```
Comment (rsandell) [inline src/main/resources/org/jenkinsci/plugins/credentialsbinding/impl/UsernamePasswordMultiBinding/help.html:5]:
```suggestion
    <strong>Warning</strong>: if the Jenkins controller or agent node has multiple executors,
```

File: src/main/resources/org/jenkinsci/plugins/credentialsbinding/impl/ZipFileBinding/help.html [hunk 7]
```diff
@@ -3,7 +3,7 @@
     (The directory is deleted when the build completes.)
 </div>
 <div>
-    <strong>Warning</strong>: if the master or agent node has multiple executors,
+    <strong>Warning</strong>: if the controller or agent node has multiple executors,
```
Comment (rsandell) [inline src/main/resources/org/jenkinsci/plugins/credentialsbinding/impl/ZipFileBinding/help.html:6]:
```suggestion
    <strong>Warning</strong>: if the Jenkins controller or agent node has multiple executors,
```

**Review by amuniz — APPROVED** [review]

**Comment by amuniz** [comment]
@jvz you are listed as maintainer in this repository, could we get a review/merge/release of this please?

**Comment by jvz** [comment]
Yeah, I thought I already approved to auto-merge this. Is that junit error legit?

**Comment by twasyl** [comment]
> Yeah, I thought I already approved to auto-merge this. Is that junit error legit?

I updated comments and help message. From reading the logs, it seems some artifacts couldn't be retrieved ending up with an `Caused by: javax.net.ssl.SSLException: Connection reset`

**Comment by jvz** [comment]
Build error appears to be jenkins-infra related. The file path is deleting the `c` in the `...\credentials-plugin` part of the filename somewhere.

**Comment by twasyl** [comment]
So can this be merged? I think auto merge won't work as the junit step is not passing.

**Comment by jvz** [comment]
There's no merge button on this repo anymore. @jglick?

**Comment by jglick** [comment]
> The file path is deleting the `c` in the `...\credentials-plugin` part of the filename somewhere.

No, that is normal, unrelated. The error can be seen in https://ci.jenkins.io/job/Plugins/job/credentials-binding-plugin/job/PR-128/7/execution/node/60/log/ and does indeed look like an infra problem @MarkEWaite  et al.:

> Cannot contact aci-maven-windows-rnr4r: java.lang.InterruptedException

I have recently run into constant problems with these agents, prompting me to shun them: https://github.com/jenkinsci/file-parameters-plugin/pull/44

**Comment by jglick** [comment]
Now the Windows build is hanging during startup in one test:

```
"TCP agent connection handler #12 with /10.244.243.248:49860" Id=573 Group=main RUNNABLE
	at java.math.BigInteger.implMulAdd(BigInteger.java:3047)
	at java.math.BigInteger.mulAdd(BigInteger.java:3018)
	at java.math.BigInteger.montReduce(BigInteger.java:2967)
	at java.math.BigInteger.implMontgomeryMultiply(BigInteger.java:2709)
	at java.math.BigInteger.montgomeryMultiply(BigInteger.java:2661)
	at java.math.BigInteger.oddModPow(BigInteger.java:2857)
	at java.math.BigInteger.modPow(BigInteger.java:2603)
	at io.github.stephenc.crypto.sscg.internal.bc.crypto.engines.RSACoreEngine.processBlock(Unknown Source)
	at io.github.stephenc.crypto.sscg.internal.bc.crypto.engines.RSABlindedEngine.processBlock(Unknown Source)
	at io.github.stephenc.crypto.sscg.internal.bc.crypto.encodings.PKCS1Encoding.encodeBlock(Unknown Source)
	at io.github.stephenc.crypto.sscg.internal.bc.crypto.encodings.PKCS1Encoding.processBlock(Unknown Source)
	at io.github.stephenc.crypto.sscg.internal.bc.crypto.signers.RSADigestSigner.generateSignature(Unknown Source)
	at io.github.stephenc.crypto.sscg.internal.bc.operator.bc.BcSignerOutputStream.getSignature(Unknown Source)
	at io.github.stephenc.crypto.sscg.internal.bc.operator.bc.BcContentSignerBuilder$1.getSignature(Unknown Source)
	at io.github.stephenc.crypto.sscg.internal.bc.cert.CertUtils.generateSig(Unknown Source)
	at io.github.stephenc.crypto.sscg.internal.bc.cert.CertUtils.generateFullCert(Unknown Source)
	at io.github.stephenc.crypto.sscg.internal.bc.cert.X509v3CertificateBuilder.build(Unknown Source)
	at io.github.stephenc.crypto.sscg.SelfSignedCertificate.generate(SelfSignedCertificate.java:169)
	at org.jenkinsci.main.modules.instance_identity.InstanceIdentity.getCertificate(InstanceIdentity.java:155)
	-  locked org.jenkinsci.main.modules.instance_identity.InstanceIdentity@55e42449
	at org.jenkinsci.main.modules.instance_identity.InstanceIdentityRSAProvider.getCertificate(InstanceIdentityRSAProvider.java:48)
	at jenkins.model.identity.InstanceIdentityProvider$KeyTypes.getCertificate(InstanceIdentityProvider.java:284)
	at jenkins.slaves.JnlpSlaveAgentProtocol4.<init>(JnlpSlaveAgentProtocol4.java:103)
```

I guess I just need to give up on testing anything on Windows.

**Comment by jglick** [comment]
⇒ #129

**Comment by jglick** [comment]
Merge in `master` and this ought to work I guess.

**Review by rsandell — APPROVED** [review]

**Comment by kerogers-cloudbees** [comment]
@jvz Can we get a release of this plugin including these changes? Thanks!

**Comment by jglick** [comment]
Still trying to get a review of #132.

**Comment by jglick** [comment]
in https://github.com/jenkinsci/credentials-binding-plugin/releases/tag/credentials-binding-1.25

**Pull request merged**